sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/LookupCatalog.scala

```diff
@@ -155,4 +155,31 @@ private[sql] trait LookupCatalog extends Logging {
         None
     }
   }
+
+  // TODO: move function related v2 statements to the new framework.
```

Comment:
@imback82 do you have time to work on this TODO?

Reply:
Yea, I will get to it once this PR is merged.

Reply:
Created #29198